scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/Scheduler.java

```diff
@@ -1288,6 +1288,23 @@ String getTaskServerLogsByTag(String id, String taskTag)
     Page<JobInfo> getJobs(int offset, int limit, JobFilterCriteria filterCriteria,
             List<SortParameter<JobSortParameter>> sortParameters) throws NotConnectedException, PermissionException;
 
+    /**
+     * Returns a list of jobs info corresponding to the given job IDs (in the same order)
+     *
+     * @param jobsId
+     *             the list of id of the jobs to return
+     *
+     * @return jobs list of the given list of id
+     * @throws NotConnectedException
+     *             if you are not authenticated.
+     * @throws PermissionException
+     *             if you have not enough permission to access this method.
+     * @throws UnknownJobException
+     *             if a job id wasn't found
```

Comment:
It should be clearer whether the `UnknownJobException` is thrown if a single job id was not found or if all jobs id of the given `jobsId` were not found.